### PR TITLE
Updated prefix for MQTT host and port

### LINKED ISSUE #80: MQTT port and host prefix not documented
After #23 the readme and the yaml templates were not updated to reflect the same change. I can provide a PR for these two files off my fork if required.

### PR DESCRIPTION
**Pull request by MatteoPiovanelli-Laser**
Closes #80 
After #23, docs hadn't been updated, as well as a couple other places affected by that change (notably yaml used in kubernetes)

### CODE REVIEW

**Review by ppatierno — CHANGES_REQUESTED** [review]

**Comment by ppatierno** [comment]
@MatteoPiovanelli-Laser Thanks for the PR I left a comment and you should also fix the DCO.

File: install/020-ConfigMap.yaml
Comment (MatteoPiovanelli-Laser) [reply install/020-ConfigMap.yaml]: I made this change. Out of interest, since since you've tagged the PR for 0.2.0, shouldn't this have been in? Will the files here be updated by the action building the release?

Comment (ppatierno) [reply install/020-ConfigMap.yaml]: It will be set here after the release, by getting what's in there the packaging folder.

File: install/020-ConfigMap.yaml
Comment (ppatierno) [inline install/020-ConfigMap.yaml:32]: this file should not have any difference, just revert the change. I see you deleted the blank line. Changing it just for a blank line would be confusing because not related to the PR itself.

**Review by ppatierno — CHANGES_REQUESTED** [review]

**Review by ppatierno — APPROVED** [review]
LGTM. Thanks for the PR!

**Comment by ppatierno** [comment]
Sorry @strimzi/maintainers I got distracted and merged the PR without any other additional approval other than mine. I hope it makes sense to you otherwise for any objections let's discuss whenever you want.

**Pull request merged**
